internal/core/command/main.go

```diff
@@ -43,9 +43,10 @@ import (
 func Main(ctx context.Context, cancel context.CancelFunc, router *mux.Router, readyStream chan<- bool) {
 	startupTimer := startup.NewStartUpTimer(internal.BootRetrySecondsDefault, internal.BootTimeoutSecondsDefault)
 
-	var useRegistry bool
+	var debugMode, useRegistry bool
 	var configDir, profileDir string
 
+	flag.BoolVar(&debugMode, "debug", false, "Turns on request/response debug logging.")
```

Comment:
Should I add this new flag to the new Common Commandline abstraction?

Reply:
Not unless you want your PR to take a dependency on this one.  

Ultimately, I think it should be.  However, I'm not sure whether the other consumers of go-mod-bootstrap have any corresponding use or implementation associated with it at this point.  Which is my way of saying, the right time to make it a generic flag is when everyone using go-mod-bootstrap has a corresponding usage/implementation for it.